### ISSUE TITLE
New plugin SocialMediaPublisher

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo#9045]
@DanielnetoDotCom Danny, I checked out the new SocialMediaPublisher plugin. Very interesting.

I would like to report the message that is displaying, in my tests:

`DataTables warning: table id=publisher_video_publisher_logsTable - Ajax error. For more information about this error, please see http://datatables.net/tn/7`

![image](https://github.com/WWBN/AVideo/assets/37880073/6ce68665-dd11-4089-a168-2fac2e9d09cc)

If you can elaborate further on how it works.
Thanks.

**Comment by DanielnetoDotCom** [comment]
this plugin is not ready to use yet, it is still under development 

![image](https://github.com/WWBN/AVideo/assets/18009293/59b0c2a5-dece-4c39-bdee-844233e50fa6)

**Comment by gbrn** [comment]
Yes, I saw. When possible, share what new features this plugin will bring to the platform.
